cmd/validate.go

```diff
@@ -0,0 +1,107 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/aerospike/asconfig/asconf"
+	"github.com/spf13/cobra"
+)
+
+const (
+	validateArgMax = 1
+)
+
+var (
+	errValidateTooManyArguments = fmt.Errorf("expected a maximum of %d arguments", convertArgMax)
+)
+
+func init() {
+	rootCmd.AddCommand(validateCmd)
+}
+
+var validateCmd = newValidateCmd()
+
+func newValidateCmd() *cobra.Command {
+	res := &cobra.Command{
+		Use:   "validate [flags] <path/to/config_file>",
+		Short: "Validate an Aerospike configuration file.",
+		Long: `Validate an Aerospike configuration file in any supported format
+				against a versioned Aerospike configuration JSON schema.
```

Comment:
```suggestion
				against a versioned Aerospike configuration schema.
```

This might confuse some. Like, "why am I using a json schema if I am using aerospike.conf"

Reply:
just an idea, could go either way

Reply:
removed it